project/FrontendBuild.scala

```diff
@@ -11,7 +11,7 @@ object FrontendBuild extends Build with MicroService {
 
   override lazy val appDependencies: Seq[ModuleID] = compile ++ test()
 
-  val compile = Seq(
+  /*val compile = Seq(
```

Comment:
Is this meant to be left in commented out?

Reply:
good spot, will remove